tools/ldb_cmd.cc

```diff
@@ -722,6 +723,36 @@ bool LDBCommand::ParseCompressionTypeOption(
   return false;
 }
 
+Status LDBCommand::MaybePopulateReadTimestamp(ColumnFamilyHandle* cfh,
+                                              ReadOptions& ropts,
+                                              Slice* read_timestamp) {
+  if (cfh->GetComparator()->timestamp_size() == 0) {
+    return Status::OK();
```

Comment:
We could consider erroring out here if the user specified `--read_timestamp` for a CF which does not have timestamps enabled.

Reply:
That's a good point. Thanks for the suggestion!